.gitignore

```diff
@@ -1,28 +1,29 @@
-*.DotSettings
-*.ncrunchsolution
-nCrunchTemp*
-*.donotcheckin
-*.suo
-*.user
-bin/
-csx/
-Bin/
-obj/
-*.Tests/lib/
-packages/
-node_modules/
-instrumented/
-*.lnk
-_ReSharper.*/
-*.vsp
-buildlog.txt
-.DS_Store
-._.DS_Store
-_NCrunch_*/
-*.ncrunchproject
-*.ReSharper
-Artifacts
-*.psess
-Blobs/
-*.xml
-NuGet/
```

Comment:
It looks like you have a line endings configuration problem that's causing uninformative diffs here.